realm/realm-library/src/syncIntegrationTest/java/io/realm/PermissionManagerTests.java

```diff
@@ -735,6 +735,93 @@ public void onError(ObjectServerError error) {
         });
     }
 
+    @Test
+    @RunTestInLooperThread
+    public void revokeOffer() {
+        // createOffer validates that the offer is actually in the __management Realm.
```

Comment:
where are you checking this "validates that the offer is actually in the __management Realm"?